### ISSUE TITLE
TypeError: Parameters to Generic[...] must all be type variables

### ISSUE THREAD

**Issue by Bluenix2** [python/typing#899]
## Descrption

I cannot get ParamSpec to work on 3.9.5:

```python
import sys
print(sys.version_info)
from typing import Generic

from typing_extensions import ParamSpec

P = ParamSpec('P')


class Testing(Generic[P]):
    """I don't get why this fails."""
    ...
```
Gives me the following output:
```powershell
PS C:\Users\%username%\Projects\wumpy> py -3.9 test.py
sys.version_info(major=3, minor=9, micro=5, releaselevel='final', serial=0)
Traceback (most recent call last):
  File "C:\Users\gabri\Projects\wumpy\test.py", line 11, in <module>
    class Testing(Generic[P]):
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python39\lib\typing.py", line 268, in inner
    return func(*args, **kwds)
  File "C:\Users\gabri\AppData\Local\Programs\Python\Python39\lib\typing.py", line 980, in __class_getitem__
    raise TypeError(
TypeError: Parameters to Generic[...] must all be type variables
```

## Version info

`pip freeze` shows `typing-extensions==3.10.0.0`.

**Comment by srittau** [comment]
`ParamSpec` is a Python 3.10 feature. You could try to import `Generic` from `typing_extensions`, but I haven't tested whether that works.

**Comment by JelleZijlstra** [comment]
3.10.0.0 is not the latest version, can you try 3.10.0.2?

**Comment by Bluenix2** [comment]
That seems to have fixed it..

I originally played around with a Jupyter notebook in Visual Studio Code to create a reproduceable example. The first thing I did was try to update but it didn't fix it, which is when I moved it into different files and tried different versions.. seems like I forgot to update the global Python 3.9.5's packages.

I'll take this as a sign to take better care of my virtual enviroments, thanks guys!
